fides-minimal/Chart.yaml

```diff
@@ -1,13 +1,11 @@
 apiVersion: v2
 name: fides-minimal
-version: 0.12.0
+version: 0.13.0
 appVersion: "2.14.0"
-description: Fides is an open-source privacy engineering platform for managing the fulfillment of data privacy requests in your runtime environment, and the enforcement of privacy regulations in your code. This version of the Helm chart removes some excess features such as the lookup function which may not be available in all cases, such as ArgoCD.
+description: Fides is an open-source privacy engineering platform for managing the fulfillment of data privacy requests in your runtime environment, and the enforcement of privacy regulations in your code. This version of the Helm chart removes some excess features such as the lookup function which may not be available in all cases.
```

Comment:
nit: but I think the example of ArgoCD was helpful, if I was using Argo this'd show up in my CTRL+F etc 😄